site/content/en/docs/tutorials/multi_node.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Using Multi-Node Clusters"
+linkTitle: "Using multi-node clusters"
+weight: 1
+date: 2019-11-24
+---
+
+## Overview
+
+- This tutorial will show you how to start a multi-node clusters on minikube and deploy a service to it.
+
+## Prerequisites
+
+- Minikube 1.9.0 (experimental) or higher
```

Comment:
* Lowercase Minikube
* This makes minikube look experimental - perhaps this should be in the title?